primer/brand
# VideoPlayer Component

## Summary

A VideoPlayer component that will be used to render hosted video content.

- https://github.com/github/primer/issues/2448

## List of notable changes:

- **added** VideoPlayer component

## What should reviewers focus on?

- Making sure the VideoPlayer performs at all screen sizes as expected.
- Stress test the controls to make sure they are performant and don't glitch

## Steps to test:

1. Open the VideoPlayer component in CI-deployed preview environment
2. Go to VideoPlayer story in Storybook
3. Verify that the VideoPlayer behaves as described in the following issue.

## Supporting resources (related issues, external links, etc):

- [API Exploration](https://www.figma.com/file/y0OnbQ7Zpku6Y1Bl7IHvkZ/VideoPlayer---API-exploration?type=whiteboard&node-id=0-1&t=FlsKh5FUvY1YMULQ-0)
- [Tracking Issue](https://github.com/github/primer/issues/2448)
- [Octicon Play Pause Issue](https://github.com/github/primer/issues/2572)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

![image](https://github.com/primer/brand/assets/31220082/81231aa8-2871-4880-8076-5b4dddd46c12)
